renderer/views/torrent-list.js

```diff
@@ -45,23 +45,18 @@ function renderTorrent (torrent, dispatch) {
 
 // Renders the torrent name and download progress
 function renderTorrentMetadata (torrent) {
-  var progressPercent = 0
-  var progressBytes = 0
+  var progress = Math.floor(100 * (torrent.progress || 0))
+  var downloaded = prettyBytes(torrent.downloaded || 0)
+  var total = prettyBytes(torrent.length || 0)
```

Comment:
`torrent.progress` and `torrent.downloaded` will never be `undefined`, so no need for `|| 0`.
The length depends on the metadata being received from the network, but I'll initialize it to 0 in webtorrent and remove it from here later.